applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,483 @@
+    }
+
+    const handleAddFile = (type: string, value: any) => {
+        setFiles([...files, value])
+        validationErrors.delete(type)
+        setValidationErrors(validationErrors)
+    }
+
+    const handlePageChange = (event: any, page: number) => {
+        setPage(page)
+    }
+
+    return populationName !== null ? (
+        <div className={classes.container} style={{ justifyContent: "space-between" }}>
+            <div>
+                <Box display="flex" justifyContent="space-between" alignItems="center" className={classes.titleBox}>
+                    <Box display="flex" alignItems="center">
+                        <span className={classes.populationColor}>
+                            <Box style={{ backgroundColor: getRGBAString(getRGBAColor()) }}
+                                component="span"
+                                className={classes.square} />
+                        </span>
+                        <Typography className={classes.title}>{populationName}</Typography>
+                    </Box>
+                    <Button variant="outlined" onClick={() => setOpenUploadDialog(true)}>Upload PDF</Button>
+                </Box>
+                <Box sx={{ bgcolor: 'transparent' }}>
+                    <Tabs value={tabIdx} onChange={handleTabChange} className={classes.tabs}>
+                        {
+                            data.map((option, index) => (
+                                <Tab key={index} disableRipple label={option.tab} />
+                            ))
+                        }
+                    </Tabs>
+                </Box>
+                <Box display="flex" alignItems="center" justifyContent="space-between" className={classes.tabPanelActions}>
+                    <Button
+                        id="detailsViewer-customized-button"
+                        className={classes.tabPanelActionsMenuBtn}
+                        aria-controls={openMenu ? 'detailsViewer-customized-menu' : undefined}
+                        aria-haspopup="true"
+                        aria-expanded={openMenu ? 'true' : undefined}
+                        variant="text"
+                        disableElevation
+                        onClick={handleMenuClick}
+                        endIcon={<img src={DOWN_ICON} alt='' />}
+                    >
+                        <span className={classes.ellipsisText}>{data[tabIdx].files[selectedIndex].title}</span>
+                    </Button>
+                    <Menu
+                        id="detailsViewer-customized-menu"
+                        MenuListProps={{
+                            'aria-labelledby': 'demo-customized-button',
+                        }}
+                        className={classes.customMenu}
+                        anchorEl={anchorElMenu}
+                        open={openMenu}
+                        onClose={handleMenuClose}
+                    >
+                        <TextField
```

Comment:
Needs a onChange prop to perform the search (can be addressed by the person responsible to connect the component)